app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt

```diff
@@ -980,7 +980,9 @@ class BrowserTabFragment :
                 builtInZoomControls = true
                 displayZoomControls = false
                 mixedContentMode = WebSettings.MIXED_CONTENT_COMPATIBILITY_MODE
-                setSupportMultipleWindows(true)
+                // There's an issue when overriding urls (GPC) and supporting blank links.
+                // Disabled until we fix the issue.
+                setSupportMultipleWindows(false)
```

Comment:
Comments explaining bugs/workarounds get confusing fast as the code around them changes.
Can we move this into a separate self-documenting method e.g "disableMultipleWindowsToSupportGPC()".

Reply:
> nit: Can you add the Asana task reference here? it will be easier to get the context back when tackling the fix

Thanks for the comments. We are enabling this again. This was just a temp proposal to release before the deadline. We have more time now and we will investigate further.